server/routerlicious/packages/services-core/src/document.ts

```diff
@@ -146,3 +146,15 @@ export interface IDocument {
 	// The document is soft deleted if a scheduled deletion timestamp is present.
 	scheduledDeletionTime?: string;
 }
+
+export interface ICheckpoint{
+    _id: string;
```

Comment:
This is very good to define here, but I don't see you set the value of `_id` anywhere, did I missed anything? Setting `_id` to documentId is very importatnt

Reply:
Added setting the value of _id when writing to the collections.